src/components/Table/Table.jsx

```diff
@@ -134,6 +136,7 @@ const propTypes = {
           columnId: PropTypes.string.isRequired,
           /* Visibility of column in table, defaults to false */
           isHidden: PropTypes.bool,
+          editDataFunction: PropTypes.bool,
```

Comment:
Am I missing something?  Shouldn't editDataFunction be a function?

Reply:
Yes, it should be a function. In the past, I wrote the contents of editDataFunction in `TableBodyRow` with a hard-code `<TextInput>` component, and if editDataFunction is set `true` in `tableColumns` of `Table.story`, the corresponding column/rows/cells will re-render into `<TextInput>` after clicking edit icon. But now, I move the contents of editDataFunction into `Table.story` and change its props as `func`, besides user can customize the contents of editDataFunction now. BTW you can see the default contents of editDataFunction is `editDefaultData` with a `<TextInput>` in `Table.story`.